### PR TITLE
Translated browser testing recipe to Spanish

### PR DESCRIPTION
**Pull request by montogeek**

### CODE REVIEW

**Comment by forresst** [comment]
cc @kutyel @AlbertoFuente @achecopar for review

File: es_ES/docs/recipes/browser-testing.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Configurar AVA para ejecutar tests en el navegador
+
+AVA no soporta ejecutar tests en el navegador, [todavía](https://github.com/avajs/ava/issues/24). Algunas librerías requieren globales especifícos para el navegador (`window`, `document`, `navigator`, etc).
+Un ejemplo de esto es React, al menos si quieres usar ReactDOM.render y simular eventos con ReactTestUtils.
+
+Esta guía funciona para cualquier librería que necesita un en torno de navegador simulado.
```
Comment (kutyel) [inline es_ES/docs/recipes/browser-testing.md]: `entorno`

**Review by kutyel — CHANGES_REQUESTED** [review]
When that single type is fixed for me the translation is perfect! 👍

File: es_ES/docs/recipes/browser-testing.md (hunk 1 above)
Comment (montogeek) [reply es_ES/docs/recipes/browser-testing.md]: Haha, so close!

**Comment by sotojuan** [comment]
Thanks!

**Pull request merged**